release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -167,6 +167,13 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} updated to HAProxy 2.6.
 
+[id="ocp-4-14-nmstate-ui-console-update"]
+==== NMstate Operator updated in console
+
+With this release, you can access NMstate Operator and resources from the web console. The Kubernetes NMState Operator provides a Kubernetes API for performing state-driven network configuration across the {product-title} cluster’s nodes with NMState.
```

Comment:
Minor comments: 
We can access **NMstate Operator** before this release, location is k8s/ns/openshift-nmstate/operators.coreos.com~v1alpha1~ClusterServiceVersion after installing it.
As per my understanding, this new feature is focus on supporting access **NMstate resources** from the web console.